modules/export_pipeline/trigger_export_pipeline.tf

```diff
@@ -0,0 +1,137 @@
+resource "google_service_account" "trigger_export" {
+  account_id   = "gcf-trigger-export-sa"
+  display_name = "Used By Trigger Export Cloud Function"
+}
+
+# Grant permissions needed for the cloud function to run and receive event triggers.
+resource "google_project_iam_member" "trigger_export_invoking" {
+  project    = data.google_project.project.project_id
+  role       = "roles/run.invoker"
+  member     = "serviceAccount:${google_service_account.trigger_export.email}"
+  depends_on = [google_project_iam_member.gcs_pubsub_publishing]
+}
+
+resource "google_project_iam_member" "trigger_export_receiving" {
+  project    = data.google_project.project.project_id
+  role       = "roles/eventarc.eventReceiver"
+  member     = "serviceAccount:${google_service_account.trigger_export.email}"
+  depends_on = [google_project_iam_member.trigger_export_invoking]
+}
+
+resource "google_project_iam_member" "trigger_export_artifactregistry_reader" {
+  project    = data.google_project.project.project_id
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${google_service_account.trigger_export.email}"
+  depends_on = [google_project_iam_member.trigger_export_receiving]
+}
+
+# Grant permissions for the cloud function to publish messages to the Pub/Sub topic.
+resource "google_project_iam_member" "trigger_export_publishing" {
+  project = data.google_project.project.project_id
+  role    = "roles/pubsub.publisher"
+  member  = "serviceAccount:${google_service_account.trigger_export.email}"
+  depends_on = [google_pubsub_topic.export_predictions_topic]
+}
+
+# Give read access to the predictions bucket.
+resource "google_storage_bucket_iam_member" "trigger_export_predictions_reader" {
+  bucket = google_storage_bucket.predictions.name
+  role   = "roles/storage.objectViewer"
+  member = "serviceAccount:${google_service_account.trigger_export.email}"
+}
+
+# Give write access to the chunk predictions bucket.
+resource "google_storage_bucket_iam_member" "trigger_export_chunk_predictions_writer" {
+  bucket = google_storage_bucket.chunk_predictions.name
+  role   = "roles/storage.objectUser"
+  member = "serviceAccount:${google_service_account.trigger_export.email}"
+}
+
+# Place the source code for the cloud function into a GCS bucket.
+data "archive_file" "source" {
+  type        = "zip"
+  output_path = "${path.module}/files/cloud_function_source.zip"
+
+  # Add main.py to the root of the zip file.
+  source {
+    content  = file("{path.module}/../../climateiq-frontend/cloud_functions/climateiq_trigger_export_pipeline_cf/main.py")
+    filename = "main.py"
+  }
+  # Add requirements.txt to the root of the zip file.
+  source {
+    content  = file("{path.module}/../../climateiq-frontend/cloud_functions/climateiq_trigger_export_pipeline_cf/requirements.txt")
+    filename = "requirements.txt"
+  }
+}
+
+resource "google_storage_bucket_object" "source" {
+  name   = basename(data.archive_file.source.output_path)
```

Comment:
looks like we're using the same name for the source .zip in both modules, so I think they'd be trying to overwrite each other here now that they're in the same bucket. I think we should set a name here rather than rely on `basename(data.archive_file.source.output_path)` being unique.